test/bpf/verifier-test.sh

```diff
@@ -6,7 +6,6 @@ set -eo pipefail
 
 DEV="cilium-probe"
 DIR=$(dirname $0)/../../bpf
-MAPTOOL=$(dirname $0)/../../tools/maptool/maptool
 # all known bpf programs (object files).
 # ALL_PROGS will be tested again source files to find non-tested bpf code
 ALL_TC_PROGS="bpf_lxc bpf_host bpf_network bpf_overlay"
```

Comment:
Can't comment on the line itself, but `sockops/bpf_sockops sockops/bpf_redir` also need to be removed from `ALL_CG_PROGS` below.